ffs/manager/manager.go

```diff
@@ -42,7 +42,7 @@ var (
 			},
 		},
 	}
-	dsDefaultCidConfigKey = datastore.NewKey("defaultcidconfig")
+	dsDefaultStorageConfigKey = datastore.NewKey("defaultcidconfig")
```

Comment:
I think it may worth adding a `// ToDo: StorageConfig renaming` (and also in other places that we're delaying this work). So to remember there's some weird/holding change, and easier to spot when doing that?